mcs/tools/mono-api-html/ApiChange.cs

```diff
@@ -4,20 +4,29 @@
 using System.Text;
 using System.Xml.Linq;
 
-namespace Xamarin.ApiDiff
-{
-	public class ApiChange
+#if USE_MONO_API_TOOLS_NAMESPACE
+namespace Mono.ApiTools {
+#else
+namespace Xamarin.ApiDiff {
+#endif
+
+#if !USE_INTERNAL_VISIBILITY
+	public
+#endif
```

Comment:
Because this tool has all it's types public, someone may be using it somewhere. So, I just have a flag for making it internal for those who don't want to see them.

Reply:
I did a quick GitHub search across the Mono and Xamarin organization and everyone seems to be using the whole tool and not just pieces so we might be able to just make this internal all the time.